### PR TITLE
Feature/add changemedia delegate

### PR DESCRIPTION
**Pull request by RealBug**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,10 @@
 All notable changes to this project will be documented in this file.
 
 ---
-## [X.X.X]
+## [0.11.0]
```
Comment (raphrel) [inline CHANGELOG.md]: keep [X.X.X], this information is changed at the release time.

File: Example/CustomTrackerPlugin.swift [hunk 2]
```diff
@@ -20,7 +20,7 @@ final class CustomTrackerPlugin: PlayerPlugin {
             else { assertionFailure(); return }
         print("~~~ PLUGIN: customAttribute=\(attr)")
     }
-    
+    func didChangeMedia(_ media: PlayerMedia, previousMedia: PlayerMedia?) { }
```
Comment (raphrel) [inline Example/CustomTrackerPlugin.swift]: didMediaChanged instead ?

File: Sources/Core/Components/PlayerPlugin.swift [hunk 3]
```diff
@@ -50,6 +50,14 @@ public protocol PlayerPlugin {
     ///
     func didLoad(media: PlayerMedia, duration: Double?)
     
+    ///
+    /// Called when the media change
```
Comment (raphrel) [inline Sources/Core/Components/PlayerPlugin.swift]: `Called when the media has changed` ?

File: Tests/Mock/MockPlayerPlugin.swift [hunk 4]
```diff
@@ -94,4 +94,13 @@ final class MockPlayerPlugin: PlayerPlugin {
         didLoadLastMediaParam = media as? MockPlayerMedia
         didLoadLastDurationParam = duration
     }
+    
+    private(set) var didChangeMediaCallCount = 0
+    private(set) var didChangeMediaParam: MockPlayerMedia?
```
Comment (raphrel) [inline Tests/Mock/MockPlayerPlugin.swift]: add Last before Param

File: Tests/Mock/MockPlayerPlugin.swift [hunk 5]
```diff
@@ -94,4 +94,13 @@ final class MockPlayerPlugin: PlayerPlugin {
...
+    private(set) var didChangePreviousMediaParam: MockPlayerMedia?
```
Comment (raphrel) [inline Tests/Mock/MockPlayerPlugin.swift]: add Last before Param

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,9 +2,10 @@
...
 * Feature:
 	* Add item to play end time information to delegate and plugin
+	* Add did change media information to plugin
```
Comment (raphrel) [inline CHANGELOG.md]: Add did media changed information to plugin

File: CHANGELOG.md (hunk 1 above)
Comment (RealBug) [reply CHANGELOG.md]: Done

File: Example/CustomTrackerPlugin.swift (hunk 2 above)
Comment (RealBug) [reply Example/CustomTrackerPlugin.swift]: Done

File: Tests/Mock/MockPlayerPlugin.swift (hunk 5 above)
Comment (RealBug) [reply Tests/Mock/MockPlayerPlugin.swift]: Done

File: CHANGELOG.md (hunk 6 above)
Comment (RealBug) [reply CHANGELOG.md]: Done

**Pull request merged**